UnitystationLauncher/Models/ServerManager.cs

```diff
@@ -60,6 +60,12 @@ private async Task<IReadOnlyList<ServerViewModel>> GetServerList()
 
             foreach (Server s in serverList.Servers)
             {
+                if (!s.HasTrustedUrlSource)
+                {
+                    Log.Warning($"Server: {s.ServerName} has untrusted download URL and has been omitted in " +
+                                "the server list!");
```

Comment:
We shouldn't interpolate the string ourselves because this makes structured logging impossible and is probably worse for performance.
Use this instead:
```suggestion
                    Log.Warning("Server: {ServerName} has untrusted download URL and has been omitted in the server list!", s.ServerName);
```

Reply:
done